File: src/game/events/game/EnemyDespawnedGameEvent.java
package game.events.game;

import game.Bomberman;
import game.events.models.GameEvent;

public class EnemyDespawnedGameEvent implements GameEvent {
    @Override
    public void invoke(Object arg) {
        if (!Bomberman.getMatch().getGameState()) return;

        Bomberman.getMatch().getCurrentLevel().onEnemyDespawned();
    }
}

File: src/game/events/game/EnemyDespawnedGameEvent.kt
package game.events.game

import game.Bomberman
import game.events.models.GameEvent

class EnemyDespawnedGameEvent : GameEvent {
    override fun invoke(arg: Any?) {
        if (!Bomberman.getMatch().gameState)
            return
        Bomberman.getMatch().currentLevel.eventHandler.onEnemyDespawned()
    }
}